documentation/source/BenchmarkingYoloNAS.md

```diff
@@ -20,24 +20,37 @@ There are plenty of guides on how to perform PTQ/QAT with SG:
 - [QA/PTQ](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/ptq_qat.md)
 
 Suppose we ran PTQ/QAT, then our PTQ/QAT checkpoints have been exported to our checkpoints directory.
-If we plug them into [netron.app](https://netron.app), we can see that new blocks that were not a part of the original network were introduced: the **Quantize/Dequantize** layers - 
+If we plug them into [netron](https://netron.app), we can see that new blocks that were not a part of the original network were introduced: the **Quantize/Dequantize** layers - 
 
 <div>
 <img src="images/qdq_yolonas_netron.png" width="750">
 </div>
+
 This is expected and an excellent way to verify that our model is ready to be converted to Int8 using Nvidia's TesnorRT.
+As stated earlier - that inference time in Pytorch is slower with such blocks - but will be faster once converted to the TRT Engine.
 
 ## Step 2: Create TRT Engine
 First, please make sure to [install Nvidia's TensorRT](https://developer.nvidia.com/tensorrt-getting-started).
 TensorRT version >= 8.4 is required.
 We can now use these ONNX files to deploy our newly trained YoloNAS models to production. When building the TRT engine, it is essential to specify that we convert to Int8 (the fake quantized layers in our models will be adapted accordingly); this can be done by running: `trtexec --fp16 --int8 --onnx=your_yolonas_qat_model. onnx.
 ## Step 3: View Model Benchmark Results
 
-Once running, `trtexec --fp16 --int8 --onnx=your_yolonas_qat_model.onnx.` your screen will look somewhat similar to the screenshot below: 
+Once running:
+```commandline
+trtexec --fp16 --int8 --avgRuns=100 --onnx=your_yolonas_qat_model.onnx.
```

Comment:
You probably don't need trailing dot symbol

Reply:
removed.